pkg/processor/runtime/rpc/connection/abstract.go

```diff
@@ -314,8 +339,6 @@ func (be *AbstractEventConnection) RunHandler() {
 				be.resultChan <- unmarshalledResults
 			case 'm':
 				be.handleResponseMetric(data[1:])
-			case 'l':
-				be.handleResponseLog(data[1:])
```

Comment:
You can't remove it from here.
Only Python runtime supports control communication and can handle logs in the control socket.
This is the abstract RPC runtime, which [doesn't support control communitcation](https://github.com/nuclio/nuclio/blob/7c8736dae1951a312a2343fb2d1f7233dcb3e12f/pkg/processor/runtime/rpc/abstract.go#L187-L190), and other runtimes [such as NodeJS](https://github.com/nuclio/nuclio/blob/103c9112f65fbe8c67b5ce3e8e04aa7b7d9cec77/pkg/processor/runtime/nodejs/js/wrapper.js#L25) still rely on this mechanism for logging.

Reply:
@TomerShor reverted + completely reworked everything